web-scrapping/Google Map Data Scraping/map_scrapper.ipynb

```diff
@@ -0,0 +1,2724 @@
+{
```

Comment:
1. Clarify the purpose of this program
2. Give some test cases
3. Give the instructions how to use it